src/instructlab/model/evaluate.py

```diff
@@ -0,0 +1,504 @@
+    """prints the base_model and model with a header"""
+    print("## BASE MODEL")
+    print(base_model)
+    display_model(model)
+
+
+def display_model(model) -> None:
+    """helper func for display_models
+    prints the given model with a header
+    """
+    print("\n## MODEL")
+    print(model)
+
+
+def display_branch_eval_summary(
+    improvements: list, regressions: list, no_changes: list, new=None
+):
+    """takes in results lists from mt_bench_branch benchmark evaluation
+    prints out diff between the branches to the user
+    """
+    if len(improvements) > 0:
+        improvements.sort(key=sort_score, reverse=True)
+        print("\n### IMPROVEMENTS:")
+        for index, improvement in enumerate(improvements):
+            task, delta = improvement
+            print(f"{index+1}. {task} (+{delta})")
+
+    if len(regressions) > 0:
+        regressions.sort(key=sort_score)
+        print("\n### REGRESSIONS:")
+        for index, regression in enumerate(regressions):
+            task, delta = regression
+            print(f"{index+1}. {task} ({delta})")
+
+    if len(no_changes) > 0:
+        print("\n### NO CHANGE:")
+        for index, task in enumerate(no_changes):
+            print(f"{index+1}. {task}")
+
+    if new is not None and len(new) > 0:
+        print("\n### NEW:")
+        for index, qna in enumerate(new):
+            print(f"{index+1}. {qna}")
+
+
+def qa_pairs_to_qna_to_avg_scores(qa_pairs: list[dict]) -> dict:
+    """takes in a list of qa_pair dicts
+    returns a dict of average scores per qna file
+    """
+    qna_to_scores = {}
+    for qa_pair in qa_pairs:
+        qna_file = qa_pair["qna_file"]
+        score = qa_pair["score"]
+        scores = qna_to_scores.get(qna_file)
+        if scores is None:
+            qna_to_scores[qna_file] = [score]
+        else:
+            scores.append(score)
+    qna_to_avg_scores = {}
+    for qna, scores in qna_to_scores.items():
+        qna_to_avg_scores[qna] = sum(scores) / len(scores)
+    return qna_to_avg_scores
+
+
+@click.command()
+@click.option(
+    "--model",
+    type=click.STRING,
+    help="Model to be evaluated - can be a local path or the name of a Hugging Face repository",
+)
+@click.option(
+    "--base-model",
+    type=click.STRING,
+    help="Base model to compare with 'model' for mt_bench_branch and mmlu_branch - can be a local path or the name of a Hugging Face repository",
+)
+@click.option(
+    "--benchmark",
+    type=click.Choice(list(BENCHMARK_TO_CLASS_MAP.keys())),
+    # case_sensitive=False,
+    help="Benchmarks to run during evaluation",
+)
+@click.option(
+    "--judge-model",
+    type=click.STRING,
+    help="Model to be used as a judge for running mt_bench or mt_bench_branch - can be a local path or the name of a Hugging Face repository",
+)
+@click.option(
+    "--output-dir",
+    type=click.Path(),
+    help="The directory to use for evaluation output from mt_bench or mt_bench_branch",
```

Comment:
don't forget to add the default `eval_results`

Reply:
The default is set to eval_data in configuration.py

Reply:
Ah. Thought it had to be added in this click arg